bin/get-individual-scoring-finesse.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import ailab.db.finesse as finesse
+import ailab.db as db
+import pandas as pd
+import dotenv
+
+# This is used to load the .env file
+dotenv.load_dotenv()
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+def create_individual_scoring(cursor):
+    """Create individual scoring view."""
+    with open("sql/2023-11-02-individual-scoring-view.sql", "r") as file:
+        sql_command = file.read()
+    cursor.execute(sql_command)
+
+if __name__ == "__main__":
+    """Get the individual scoring of each documents and write the results in a csv file."""
+    connection = db.connect_db()
+    with db.cursor(connection) as cursor:
+        create_individual_scoring(cursor)
+        results = finesse.get_individual_scoring(cursor)
+    file = 'tests/output/individual_scoring.csv'
+    dataframe = pd.DataFrame(results)
+    logger.info("Creating " + file)
+    dataframe.to_csv(file, index=False)
+    
```

Comment:
you fixed the eof problem in one place but not in every file. usually if I point out the problem once you need to check that it's not also present elsewhere.

Reply:
I've put the setting thingy to add an eof but it seems like it's not working every time. I'll check every file and commit the changes.

Reply:
done

Reply:
please check with the Github web pull request interface; there's still the marker for some of these files meaning the last character

![image](https://github.com/ai-cfia/ailab-db/assets/538542/64fbb427-42b4-469c-a614-6bb32f0e8ee7)

Reply:
adding a line manually with an indent at the end of the file doesn't work:

![image](https://github.com/ai-cfia/ailab-db/assets/538542/0b1535c4-e473-489f-8d40-7106361319a3)

the last character needs to be a new linefeed on its own.

Reply:
should be good this time